java/sage/media/format/FormatParser.java

```diff
@@ -395,6 +423,46 @@ private static boolean addAdditionalMetadata(java.io.File f, ContainerFormat for
     return rv;
   }
 
+  private static FormatParserPlugin getFormatParserPluginInstance()
+  {
+    /*
+     * Check to see if an instance has already been created.  If not check to see if one is configured and attempt to create an instance
+    */
+    if (FormatParser.formatParserPluginInstance == null)
+    {
+      String parsePlugin = sage.Sage.get("mediafile_mediaformat_parser_plugin", "");
+
+      if (parsePlugin.isEmpty())
+      {
+        FormatParser.formatParserPluginInstance = null;
+      }
+      else
+      {
+        synchronized (formatParserPluginLock)
+        {
+          if (FormatParser.formatParserPluginInstance != null)
```

Comment:
Remove the FormatParser. prefix here and the next few lines below and in the return statement.